decidim-core/app/packs/src/decidim/redesigned_input_character_counter.js

```diff
@@ -0,0 +1,313 @@
+/* eslint max-lines: ["error", {"max": 350, "skipBlankLines": true}] */
+
+const COUNT_KEY = "%count%";
+// How often SR announces the message in relation to maximum characters. E.g.
+// if max characters is 1000, screen reader announces the remaining characters
+// every 100 (= 0.1 * 1000) characters. This will be "floored" to the closest
+// 100 if the maximum characters > 100. E.g. if max characters is 5500, the
+// threshold is 500 (= Math.floor(550 / 100) * 100). With 100 or less
+// characters, this ratio is omitted and the announce threshold is always set to
+// 10.
+const SR_ANNOUNCE_THRESHOLD_RATIO = 0.1;
+// The number of characters left after which every keystroke will be announced.
+const SR_ANNOUNCE_EVERY_THRESHOLD = 10;
+const DEFAULT_MESSAGES = {
+  charactersAtLeast: {
+    one: `at least ${COUNT_KEY} character`,
+    other: `at least ${COUNT_KEY} characters`
+  },
+  charactersLeft: {
+    one: `${COUNT_KEY} character left`,
+    other: `${COUNT_KEY} characters left`
+  }
+};
+let MESSAGES = DEFAULT_MESSAGES;
+
+export default class InputCharacterCounter {
+  static configureMessages(messages) {
+    MESSAGES = $.extend(DEFAULT_MESSAGES, messages);
+  }
+
+  constructor(input) {
+    this.$input = input;
+    this.$target = $(this.$input.data("remaining-characters"));
+    this.minCharacters = parseInt(this.$input.attr("minlength"), 10);
+    this.maxCharacters = parseInt(this.$input.attr("maxlength"), 10);
+    this.describeByCounter = typeof this.$input.attr("aria-describedby") === "undefined";
+
+    // Define the closest length for the input "gaps" defined by the threshold.
+    if (this.maxCharacters > 10) {
+      if (this.maxCharacters > 100) {
+        this.announceThreshold = Math.floor(this.maxCharacters * SR_ANNOUNCE_THRESHOLD_RATIO / 100) * 100;
```

Comment:
In the technical review of the bugfixes at #9680, @andreslucena still found one issue that I fixed at 56fffb3d5b73755d14cf768ea42459411762f30a.

Could you please apply the same fix here?